src/js/modules/accounts/components/ProfilePage.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import { Link } from "react-router-dom";
+import injectSheet from "react-jss";
+
+import { SignOutForm } from "./forms";
+import { signOut } from "../actions";
+
+const styles = {
+  successMessage: {
+    color: "green",
+  },
+};
+
+const ProfilePage = ({ classes, session, signOut, status, users }) => {
```

Comment:
I'm guessing status is a redux form thing?

Reply:
I'd consider splitting up some of these things. It's good code, but it takes a little bit to read each of the blocks and figure out what they do. It'd be better if there were just named components